### PR TITLE
Gh 3055/new intercom api key

### PR DESCRIPTION
**Pull request by biafra23**
Handles #3055

Changes proposed in this pull request:
- Changed URLs in Config.example.xcconfig
- Added region key to Info.plist
- Credentials in the CI have been updates
- **ATTENTION** If for some reason we want to update the App store app without the changes in this PR, then we need to go back to the previous intercom credentials as well.

### CODE REVIEW

File: Multisig/Info.plist [hunk 1]
```diff
@@ -95,6 +95,8 @@
 	<false/>
 	<key>IntercomAutoIntegratePushNotifications</key>
 	<string>NO</string>
+	<key>IntercomRegion</key>
```
Comment (MouazAlzahabi) [inline Multisig/Info.plist:98]: Can you help me understand why this change required ?

**Review by MouazAlzahabi — APPROVED** [review]

**Review by elgatovital — APPROVED** [review]

File: Multisig/Info.plist (hunk 1 above)
Comment (elgatovital) [reply Multisig/Info.plist:98]: We are migrating Intercom from us to eu region

Comment (elgatovital) [reply Multisig/Info.plist:98]: https://developers.intercom.com/installing-intercom/docs/ios-data-hosting-region-configuration
